sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/LiveResourceManager.cs

```diff
@@ -25,7 +25,7 @@ namespace Azure.Messaging.EventHubs.Tests
     public sealed class LiveResourceManager
     {
         /// <summary>The maximum number of attempts to retry a management operation.</summary>
-        private const int RetryMaximumAttemps = 15;
+        private const int RetryMaximumAttemps = 20;
```

Comment:
With more libraries moving to using dynamic resources, ARM has gotten more cranky and we've seen an uptick in intermittent failures related to creating Azure resources.